ApplicationPattern.yaml

```diff
@@ -476,15 +475,14 @@ paths:
                           type: string
                           description: >
                             'Label that shall be presented on the button
-                            from [/core-model-1-4:control-construct/profile-collection/profile=*-1-0-0-action-p-000/action-profile-1-0:action-profile-pac/action-profile-capability/label'
+                            from [/core-model-1-4:control-construct/profile-collection/profile=*-1-0-0-action-p-*/action-profile-1-0:action-profile-pac/action-profile-capability/label'
```

Comment:
A small typo in description of "label" parameter. A closing bracket '**]**' is missing at end.

Reply:
Added the ']'. Thanks for indicating.